src/primary-navigation-layout/terra-application-navigation/header/_Header.jsx

```diff
@@ -192,9 +188,6 @@ const Header = ({
 
   return (
     <header role="banner" className={cx('header')}>
-      <button type="button" role="link" onClick={onSelectSkipToContent} className={cx('skip-content-button')}>
```

Comment:
SkipTo links will be a feature of the ApplicationContainer and added with a separate story.